<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Titlewg</class>
 <widget class="QWidget" name="Titlewg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="logo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="app_name">
     <property name="font">
      <font>
       <family>Copperplate Gothic Bold</family>
      </font>
     </property>
     <property name="text">
      <string>MARS' CLOUD</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>942</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="min_btn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/rsc/icons8-apple-tv-16.png</normaloff>:/rsc/icons8-apple-tv-16.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="set_btn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/rsc/icons8-apple-pay-16.png</normaloff>:/rsc/icons8-apple-pay-16.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="close_btn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/rsc/icons8-apple-logo-16.png</normaloff>:/rsc/icons8-apple-logo-16.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
